app/assets/javascripts/misc/verify-modal.js

```diff
@@ -0,0 +1,42 @@
+const I18n = window.LoginGov.I18n;
```

Comment:
browserify, the library we use to compile our JS, puts each file into its own function context. This way, no variables are leaked into the global scope beyond those explicitly put on the `window` object.